css-brochure-form/styles.css

```diff
@@ -0,0 +1,62 @@
+h1 {
+  text-align: center;
+  color: #f07167;
+  font-family: AtlasGrotesk, "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
+  text-transform: uppercase;
+}
+
+body {
+  text-align: center;
+}
+
+section {
+  font-family: "Brandon Grotesque", AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+input.first,
```

Comment:
I recommend setting an appropritate width on the labels to reduce the amount of padding you are using. padding, like margin, should be used in small mounts.